.github/workflows/python.yml

```
@@ -21,47 +21,19 @@ jobs:
-        python-version: [ '3.9', '3.10', '3.11', '3.12', '3.13' ]
-        include:
-          - os: ubuntu-latest
-            pip-cache: ~/.cache/pip
-            poetry-cache: ~/.cache/pypoetry
-          - os: macos-13
-            pip-cache: ~/Library/Caches/pip
-            poetry-cache: ~/Library/Caches/pypoetry
-          - os: windows-latest
-            pip-cache: ~\AppData\Local\pip\Cache
-            poetry-cache: ~\AppData\Local\pypoetry\Cache
+        os: [ubuntu-latest, macos-13, windows-latest]
```

actionlint (1.7.12)

[error] label "macos-13" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)